.github/workflows/pull_request.yml

```diff
@@ -56,6 +46,49 @@ jobs:
       - name: Install yarn deps
         if: steps.cache-yarn.outputs.cache-hit != 'true'
         run: yarn
+  lint:
+    needs: python-dependencies
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          cache: 'pipenv'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14.19.0"
+      - name: Write app version
+        run: printf "${{ github.event.pull_request.head.sha }}" > .application-version
+      - name: Install pipenv
+        run: pip install pipenv==2022.11.30
+      - name: Install virtual environment
+        run: pipenv install --dev
+      - name: Load templates
+        run: make load-design-system-templates
+      - name: Compile translations
+        run: make translate
+      - name: Running translation tests
+        run: pipenv run python -m scripts.extract_translation_templates --test
+      - name: Running lint tests
```

Comment:
Worth updating this and the JS to be similar, maybe just `Python liniting` and `Javascript linting`?

Reply:
Changed them now, thanks.